### PR TITLE
[Do not merge] Release/1.6.x

### PR DESCRIPTION
**Pull request by Roman-Kornev**
## PR Checklist:

- [ ] I have implemented all requirements? (see JIRA, project documentation).
- [ ] I am not affecting someone else's work, If I am, they are included as a reviewer.
- [ ] I have added relevant tests (unit, integration or regression).
- [ ] I have added comments to all the bits that are hard to follow.
- [ ] I have added/updated Documentation.
- [ ] I have updated the appropriate changelog with a line for my changes.

### CODE REVIEW

File: packages/dara-components/changelog.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 title: Changelog
 ---
 
+## NEXT
```
Comment (krzysztof-causalens) [inline packages/dara-components/changelog.md:5]: Changelog should be updated to the correct version number

File: packages/dara-components/package.json [hunk 2]
```diff
@@ -52,7 +52,7 @@
   },
   "dependencies": {
     "@bokeh/bokehjs": "~3.1.1",
-    "@darajs/core": "1.6.3",
+    "@darajs/core": "1.6.4",
     "@darajs/styled-components": "~1.5.2",
```
Comment (krzysztof-causalens) [inline packages/dara-components/package.json:56]: UI versions should be on 1.5.4

File: packages/dara-components/changelog.md (hunk 1 above)
Comment (Roman-Kornev) [reply packages/dara-components/changelog.md:5]: Isn't it updated as part of CI?

Comment (krzysztof-causalens) [reply packages/dara-components/changelog.md:5]: Same as in the ui repo, the CI does not modify any files as it does not have the privileges to commit changes.

Comment (Roman-Kornev) [reply packages/dara-components/changelog.md:5]: Right, it only updates it in the artifact

**Comment by krzysztof-causalens** [comment]
Releasing v1.6.5, includes
-  #215

**Review by tamara-causalens — APPROVED** [review]

**Comment by krzysztof-causalens** [comment]
v1.6.7 includes
- #225 
- #228

**Review by tamara-causalens — APPROVED** [review]

**Pull request closed without merging**